client/src/profile/ProfileContainer.js

```diff
@@ -95,16 +95,18 @@ class ProfileContainer extends React.Component {
 }
 
 ProfileContainer.propTypes = {
-  profile: PropTypes.object,
-  user: PropTypes.object.isRequired,
+  accessToken: PropTypes.string,
+  email: PropTypes.string,
```

Comment:
Why is this no longer required? It's still required in the UserForm so we are still going to get an error here.